plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/store/WCOrderStore.kt

```diff
@@ -408,6 +408,14 @@ class WCOrderStore @Inject constructor(
         return ordersDao.getOrderForIdSet((orderIdentifier.toIdSet()))
     }
 
+    /**
+     * Given an [RemoteId] and [SiteModel],
+     * returns the corresponding order from the database as a [WCOrderModel].
+     */
+    fun getOrderByRemoteIdAndSite(orderRemoteId: RemoteId, site: SiteModel): WCOrderModel? {
```

Comment:
As `WCOrderStore` is a class exposed to client and in client we agreed to not use `RemoteId` for `Order` identification, WDYT about changing signature to accept primitive?

I would see benefit of `RemoteId` here if this method would be used in `FluxC` but I think it's not the case here.

```suggestion
    fun getOrderByRemoteIdAndSite(orderRemoteId: Long, site: SiteModel): WCOrderModel? {
```

Reply:
Good point! Done - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2225/commits/da3c3c237e74e77d25b07fc1e6cb640997ac7b14